ClosedXML/Excel/XLWorkbook_Load.cs

```diff
@@ -1401,7 +1401,7 @@ private void LoadClientData<T>(IXLDrawing<T> drawing, XElement clientData)
             LoadDrawingProtection<T>(drawing, clientData);
 
             var visible = clientData.Elements().FirstOrDefault(e => e.Name.LocalName == "Visible");
-            drawing.Visible = visible != null && visible.Value.ToLower().StartsWith("t");
+            drawing.Visible = visible != null && !string.Equals("false", visible.Value, StringComparison.OrdinalIgnoreCase);
```

Comment:
This is kind of undoing what was done in e7b0a78f465790aecf9ab0a9f395a3283490f208 . I'm not 100% certain, but it seems this was to support both `true`/`false` and `t`/`f` values.

Reply:
Or do you think we should just continue with your approach? All our tests are passing, so one day, when someone gives us a real test case that fails here, we can add support.

Reply:
I think I can change this to accept both f and false, but no value has to be treated as true (unlike the previous version) otherwise visible comments disappear.